pkg/storkctl/migration.go

```diff
@@ -450,6 +458,26 @@ func updateIBPObjects(kind string, namespace string, activate bool, ioStreams ge
 		}
 	}
 }
+func updateVMObjects(kind string, namespace string, activate bool, ioStreams genericclioptions.IOStreams) {
```

Comment:
Explanation comment? 
It's not clear how this relates to the qps part of the PR

Reply:
It is part of this change related to VMs - https://github.com/libopenstorage/stork/pull/1095
I believe @ram-infrac did all the changes together and all storkctl changes ended up in this PR.